src/promql/src/planner.rs

```diff
@@ -641,7 +641,10 @@ impl PromPlanner {
             }
         }
         let matchers = matchers.into_iter().collect();
-        Ok(Matchers { matchers })
+        Ok(Matchers {
```

Comment:
Maybe `Matchers::new(matchers)`? The `or_matchers` field is only used in [VictoriaMetrics](https://victoriametrics.com/) context, and there is no need to be aware of this field so far.

Reply:
Thank you. I made the trivial chanegs to let it compiled. Unaware of `Matchers::new(matchers)` before. Let me update it.